### PR TITLE
Add the Discord FAQ into the website

### PR DESCRIPTION
**Pull request by specterflare**
I saw that #10 was only missing the Discord FAQ, so I added it. At last, the consolidation is complete! The reason why so little actually ended up being added is that most of it was already here or was actually a guide, not an FAQ. I also fixed a few grammar issues.

### CODE REVIEW

File: src/help/faq/README.md [hunk 1]
```diff
@@ -313,30 +340,32 @@ To solve this issue, follow the below steps.
...
-**Mangakakalot**/**Manganelo** has created new entries for manga again on their site. Some old entries are obsolete and will not work. To resolve this, [migrate](/help/guides/source-migration/#source-migration) the manga From **Mangakakalot**/**Manganelo** source to another source.
+**Mangakakalot**/**Manganelo** have created new entries for manga on their sites again. Some old entries are obsolete and will not work. To resolve this, [migrate](/help/guides/source-migration/#source-migration) the manga from **Mangakakalot**/**Manganelo** to another source.
 
 #### What does `Source URL has changed` mean?
 
 This means that the manga is dead and needs to be migrated from **Mangakakalot**/**Manganelo** to a different source. Refer to the guide for **Page list is empty** above.
 
 #### How do I fix the `Image could not be decoded` error?
-Open manga in Webview and wait for the image to comes up. If the image isn't loading you will need to look elsewhere to read that particular manga.
+Open manga in Webview and wait for the image to comes up. If the image isn't loading you will need to look elsewhere to read that particular manga, as Mangakakalot is now blocking entire regions.
```
Comment (GHOSTBEAR) [inline src/help/faq/README.md:353]: To me, this sounds like only Mangakakalot has this issue now. Maybe make Mangakakalot as an example or try to rephrase it

File: src/help/faq/README.md [hunk 2]
```diff
@@ -313,30 +340,32 @@ To solve this issue, follow the below steps.
...
 
 ### Webtoons
 
 #### How do I fix the `Page list is empty` error?
 You will have to open the manga entry in WebView and verify your age, as Webtoons has recently added age verifications on their site.
 
-### Why are some Japscan chapters no loading?
+### Japscan
+
+### Why are some Japscan chapters not loading?
```
Comment (GHOSTBEAR) [inline src/help/faq/README.md:362]:
```suggestion
#### Why are some Japscan chapters not loading?
```
This is a subtitle of Japscan right?

Comment (specterflare) [reply src/help/faq/README.md:362]: Yeah, nice catch.

File: src/help/faq/README.md (hunk 1 above)
Comment (specterflare) [reply src/help/faq/README.md:353]: Could you rephrase that? I don't quite understand what you're saying.

Comment (GHOSTBEAR) [reply src/help/faq/README.md:353]: Okay, I just saw that this is under `Mangabox`. But the Discord FAQ isn't stating that Mangakakalot is the problem child, right? I'm not against adding it because it would let users know what causes it, but would any other extension in Mangabox also throw this error?

Comment (specterflare) [reply src/help/faq/README.md:353]: Oh, gotcha. Yeah, I'll add that.

**Comment by specterflare** [comment]
If this does get merged, will the FAQ channel in the Discord be deleted or stay up?

**Comment by GHOSTBEAR** [comment]
> If this does get merged, will the FAQ channel in the Discord be deleted or stay up?

I don't know the answer to this question maybe @Soitora or any other staff member on Discord knows.
But my guess is that it will stay up, maybe they will slightly modify it
